### ISSUE TITLE
после добавления нового местоположения на главной странице стал показывать только яндекс.карту

### ISSUE THREAD

**Issue by arbitr38** [maxp/meteo38#4]

**Comment by maxp** [comment]
Согласен, что эта часть не интуитавно понятна. Но сейчас это работает так - 
на сайте показываются только те станции, которые были выбраны в настройках (по всей видимости вы выбрали одну), если по ним были данные в течение часа.
В настройках видны станции, котороые работали в течение недели.

Если в настройки не заходили, то показывается дефолтный список.

Попробуйте пометить в настройках еще какие-нибудь станции.
